packages/blockchain-api/src/transaction/TransfersNavigator.ts

```diff
@@ -85,6 +85,20 @@ export class TransfersNavigator {
     )
   }
 
+  isEscrowReceivedToEOA(senderAddress: Contracts | string): boolean {
+    return this.transferCollection.length === 1 && this.containsTransferFrom(senderAddress)
+  }
+
+  isEscrowReceivedToMTW(senderAddress: Contracts | string): boolean {
+    const transfer = this.getTransferFrom(senderAddress)!
```

Comment:
If we move this method to `EscrowReceived`, we can make names of these variables more readable.
```
const transferToAcccount = this.getTransferFrom(Contracts.Escrow)
const transfertoWallet = this.getTransferFrom(transferToAccount.toAddressHash) 
```